pkgs/development/python-modules/boddle/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, bottle
+}:
+
+buildPythonPackage rec {
+  pname = "boddle";
+  version = "0.2.9";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "keredson";
+    repo = pname;
+    rev = "74413e4c5b39a37ea1694fab07b459ccd2998390";
```

```suggestion
    # upstream doesn't tag releases
    rev = "74413e4c5b39a37ea1694fab07b459ccd2998390";
```

Suggest you open an issue upstream to suggest they start doing this too. There are a few reasons we dislike pointing at specific commit ids:
 - it's not very transparent - there's no guarantee that the commit you're pointing at is even in one of upstream's branches
 - it doesn't work with auto-updaters